dp_wizard/utils/argparse_helpers.py

```diff
@@ -1,18 +1,7 @@
 import argparse
-from pathlib import Path
 from sys import argv
 from typing import NamedTuple
 
-
-def _existing_csv_type(arg: str) -> Path:
-    path = Path(arg)
-    if not path.exists():
-        raise argparse.ArgumentTypeError(f"No such file: {arg}")
-    if path.suffix != ".csv":
-        raise argparse.ArgumentTypeError(f'Must have ".csv" extension: {arg}')
-    return path
```

Comment:
Unused! This goes back to when input files could be specified on the command line.